prov/rxm/src/rxm.h

```diff
@@ -133,7 +133,7 @@ struct rxm_ep_wire_proto {
 	uint8_t	ctrl_version;
 	uint8_t	op_version;
 	uint8_t endianness;
-	uint8_t padding[6];
+	uint8_t padding[5];
```

Comment:
Why did the padding change here?

Reply:
The padding that was previously specified was wrong in that it pushed the element alignment into the next 32-bit word, not a 32-bit word boundary. This save a word of CM private data space and we used it in implementing the XRC connection management in CM private data.